centreon/www/install/php/Update-next.php

```diff
@@ -240,6 +240,17 @@
     }
 };
 
+/** -------------------------------------- Backup updates -------------------------------------- */
+$setBackupMysqlConfDefaultAsEmpty = function () use ($pearDB, &$errorMessage): void {
+    $errorMessage = 'Unable to reset default of database configuration path in backup configuration';
+    $pearDB->update(
```

Comment:
```suggestion
    $errorMessage = 'Unable to reset default of database configuration path in backup configuration';
    CentreonLog::create()->info(
            logTypeId: CentreonLog::TYPE_UPGRADE,
            message: "UPGRADE - {$version}: Resetting database configuration path to default value on 'options' table",
        );
    $pearDB->update(
```

...or whatever meaningful log you think would be relevant.